.eslintrc.json

```diff
@@ -21,5 +21,7 @@
         "@typescript-eslint"
     ],
     "rules": {
+//        "no-unused-vars": "off"
```

Comment:
Dlaczego? To dobra zasada

Reply:
dodałem to sam do swojego lokalnego repo i komentuje jak robie push na remote, nie wiedziałem czy to legalna zasada :)